HACKING.md

```diff
@@ -8,7 +8,7 @@
    with no HyperShift installed yet.
 
 2. Build HyperShift.
-
+        # requires go v1.16+
```

Comment:
@bparees if you look at a preview of this page, this doesn't look right. Maybe the requires comment should go in the line above where it is.

Reply:
fixed